### PR TITLE
Updating kaushalbuccha.json

### PR DESCRIPTION
**Pull request by kaushalbuccha**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/kaushalbuccha.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "description": "Devloper Portfolio",
+    "repo": "https://github.com/kaushalbuccha/Portfolio",
+    "owner": {
+        "username": "kaushalbuccha",
+        "email": "kaushaljain741@gmail.com",
+        "twitter": "kaushalbuccha"
+    },
+    "record": {
+        "A": [
+            "75.2.60.5"
```
Comment (CuteDog5695) [inline domains/kaushalbuccha.json:11]: Please give a site preview.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/kaushalbuccha.json (hunk 1 above)
Comment (kaushalbuccha) [reply domains/kaushalbuccha.json:11]: https://kaushalbuccha.netlify.app/

Comment (kaushalbuccha) [reply domains/kaushalbuccha.json:11]: > Please give a site preview.

https://kaushalbuccha.netlify.app/

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**